deploy/charts/rawfile-csi/templates/controller/service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
What is this needed for?

Reply:
It was there before, I think it used for monitoring, admission web hook etc.